### PR TITLE
fix(shared-components): solve update issues

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Fixed portal shared components version upgrade issues

## Why

To reflect shared components changes

## Issue

NA

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Review by lavanya-bmw — COMMENTED** [review]
@evegufy dependencies check is failing

**Comment by evegufy** [comment]
IP Issues have been created:
A review is required for npm/npmjs/@mui/x-data-grid/6.18.2.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11696 .
A review is required for npm/npmjs/-/react-i18next/13.5.0.
A review request was created https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/11697 .

**Comment by manojava-gk** [comment]
@oyo / @evegufy Kindly review this PR on priority. As Lavanya is on leave, could you update PR title here? I do not have access to do it

**Comment by evegufy** [comment]
@manojava-gk I checked and the local build fails. Could you please solve it?

**Comment by manojava-gk** [comment]
@oyo  Its ready now

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +24,8 @@
   - Enable return-await rule
   - Enable prefer-reduce-type-parameter rule
   - Enable prefer-readonly rule
+  - Enable no-inferrable-types rule
+  - Enable dot-notation rule
```
Comment (evegufy) [inline CHANGELOG.md:28]: were those changes to the changelog intended?

**Review by evegufy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (manojava-gk) [reply CHANGELOG.md:28]: Effect of conflict resolve. This section was in different line, I just moved them to one section.

**Review by evegufy — APPROVED** [review]
lgtm, also local build works @oyo could you please review as well?

**Review by oyo — APPROVED** [review]
lgtm

**Pull request merged**